File: Wirtualny_Swiat_Java/Projekt_PO_Java/src/model/Organisms/Sheep.java
package model.Organisms;

import model.Animal;
import model.Point;
import model.World;

import java.awt.*;

public class Sheep extends Animal {
    private Color sheepColor = Color.LIGHT_GRAY;

    public Sheep(World world, Point position){
        super(world, position, 4, 4);
    }

    @Override
    public Color getColor(){
        return sheepColor;
    }

    @Override
    public String getNazwa(){
        return "Owca";
    }
}

File: WirtualnySwiatPython/Animals/Sheep.py
from animal import Animal
from config import LIGHT_GRAY

class Sheep(Animal):
    def __init__(self, world, position):
        super().__init__(world, position, power=4, initiative=4)
        self.color = LIGHT_GRAY

    def getColor(self):
        return self.color

    def getNazwa(self):
        return "Owca"

    def makeNewOrganism(self, position):
        return Sheep(self.world, position)